Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -281,6 +321,114 @@ public bool ExecuteDataToJira(BusinessFlow bizFlow, PublishToALMConfig publishTo
+                                                                                                                                        ALMCore.DefaultAlmConfig.ALMServerURL, string.Empty, string.Empty,
+                                                                                                                                        ALMCore.DefaultAlmConfig.ALMProjectKey, bizFlow.ExternalID, string.Empty, string.Empty,
+                                                                                                                                        string.Empty, string.Empty, bizFlow.ExternalID2 == null ? string.Empty : bizFlow.ExternalID2).DataResult;
+                    if ((executionList != null) && (executionList.Count > 0))
+                    {
+                        ZephyrExecutionStatus zephyrStepStatus;
+                        ZephyrExecutionStatus zephyrIssueStatus;
+                        if (string.IsNullOrEmpty(publishToALMConfig.VariableForTCRunName))
+                        {
+                            String timeStamp = DateTime.Now.ToString("dd-MMM-yyyy HH:mm:ss");
+                            publishToALMConfig.VariableForTCRunName = "GingerRun_" + timeStamp;
+                        }
+                        foreach (var actGroup in bizFlow.ActivitiesGroups)
+                        {
+                            List<Activity> activities = (bizFlow.Activities.Where(x => x.ActivitiesGroupID == actGroup.Name)).Select(a => a).ToList();
+                            JiraZephyrExecution currentActivitiesGroupExecution = executionList.Where(z => z.IssueId.ToString() == actGroup.ExternalID).FirstOrDefault();
+                            if (currentActivitiesGroupExecution != null)
+                            {
+                                List<JiraZephyrStepResult> stepResults = ((JiraManagerZephyr)jiraRepObj.TestAlmManager()).GetZephyrStepResultsList(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword,
+                                                                                                                                                ALMCore.DefaultAlmConfig.ALMServerURL, currentActivitiesGroupExecution.Id).DataResult;
+                                foreach (var act in activities)
+                                {
+                                    zephyrStepStatus = ConvertGingerStatusToZephyr(act.Status.HasValue ? act.Status.Value : eRunStatus.NA);
+                                    JiraZephyrStepResult currentStepResult = stepResults.Where(z => z.StepId.ToString() == act.ExternalID).FirstOrDefault();
+                                    if (currentStepResult != null)
+                                    {
+                                        currentStepResult = ((JiraManagerZephyr)jiraRepObj.TestAlmManager()).UpdateZephyrStepResult(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword,
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'currentStepResult'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)